remote.c

```diff
@@ -2473,7 +2547,23 @@ do_print_cookie_secrets(RES* ssl, xfrd_state_type* xrfd, char* arg) {
 	int i;
 	(void)arg;
 
-	/* (void)ssl_printf(ssl, "cookie_secret_count=%zu\n", nsd->cookie_count); */
+	switch(nsd->cookie_secrets_source){
+	case COOKIE_SECRETS_NONE:
+		break;
+	case COOKIE_SECRETS_GENERATED:
+		ssl_printf(ssl, "source : random generated\n");
+		break;
+	case COOKIE_SECRETS_FROM_FILE:
+		ssl_printf( ssl, "source : \"%s\"\n"
+		          , nsd->cookie_secrets_filename);
+		break;
+	case COOKIE_SECRETS_FROM_CONFIG:
+		ssl_printf(ssl, "source : configuration\n");
+		break;
+	default:
+		ssl_printf(ssl, "source : unknown\n");
+		break;
```

Comment:
All possibilities are covered, so this is a panic situation(?)

Reply:
I guess yes. Would you prefer a assert here?

Reply:
Would you prefer an assert?